src/components/common/empty_content.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { HTMLAttributes } from 'react';
+import styled from 'styled-components';
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+    alignAbsoluteCenter?: boolean;
```

Comment:
The two usages of this component set this to `true`. Are we anticipating a component that needs to use this with `false`?

Reply:
Yes, you might not need (or not want to) to center it occupying all of its container, or you may not have a parent with a `position: relative;` property (which could be disastrous).